styles.css

```
@@ -0,0 +1,41 @@
+
+
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200&display=swap');
+
+body{
+  font-family:"poppins";
+  line-height: 1.5;
+  background-color: #FBEEC1;
+  width: 450px;
+  margin: auto;  
+}
+form{
+  background: rgb(204,255,255);
+  padding: 2.5rem 2.5rem;
+  border-radius: 2rems;
+  width: 500px;
+  margin: auto;
+}
+.submit-button {
+  width:450px;
+  padding: 0.75rem;
+  background: blue;
+  color:white;
+  border-radius: 2px;
+  cursor:pointer;
+}
+.input-headings {
+  margin: o auto 1.25rem auro;
```

I think you made mistakes using auro
```suggestion
  margin: o auto 1.25rem auto;
```

---

- [ ] Please consider asking for changes in a polite way, and always state to the student the reason behind your change request so it won't feel unfair to the student 👍🏼